site/pt/beta/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>I believe there are a few grammar mistakes. The pronoun "Ele", which refers to "base de dados", should be in the feminine form, since it referrs to a feminine noun. Despite that, "avaliações" is a plural noun and there are a few verbal disagreements when referring to it:</p><blockquote>A base de dados vem empacotada com TensorFlow. <strong>Ela</strong> já vem <strong>pré-processada</strong> de forma que as avaliações (sequências de palavras) <strong>foram</strong> <strong>convertidas</strong> em sequências de inteiros, onde cada inteiro representa uma palavra específica no dicionário.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>